### PR TITLE
feat(external-api): expose config for breakout rooms

### PR DESCRIPTION
**Pull request by mihhu**
Add `hideJoinRoomButton` and move `hideAddRoomButton` under the common key (backwards compatibility included).

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -476,6 +476,9 @@ var config = {
     // Hides add breakout room button
     // hideAddRoomButton: false,
 
+    // Disables join breakout room for non-moderator participants.
+    // joinRoomModeratorOnly: false,
```
Comment (saghul) [inline config.js:480]: This option has no "breakout" in the name, so it'd weird to see out of context. I realize that's also the case with the previous one 😢 

I'd say it's time to add a `breakoutRooms` section where we put the options (remember to add backwards compatibility).

File: config.js [hunk 2]
```diff
@@ -476,6 +476,9 @@ var config = {
     // Hides add breakout room button
     // hideAddRoomButton: false,
 
+    // Disables join breakout room for non-moderator participants.
```
Comment (saghul) [inline config.js:479]: Why not just  an option to hide the join button period? This way API users can decide. What if not all moderators should see it, but just the meeting "owner" ?

**Review by saghul — CHANGES_REQUESTED** [review]
Left some comments, PTAL.

File: config.js (hunk 2 above)
Comment (mihhu) [reply config.js:479]: It felt more consistent with the rest of the implementation. Maybe we could provide a distinct flag for hiding the button, but I'm not sure how to handle the owner case.

Comment (saghul) [reply config.js:479]: How so? The `hideAddRoomButton` makes no distinction.

File: config.js (hunk 1 above)
Comment (saghul) [reply config.js:480]: No :-) We generally do it as we add the new option, otherwise we need to immediately add backwards compat code, which is not ideal.

**Review by saghul — APPROVED** [review]

**Pull request merged**